src/rtext.c

```diff
@@ -1766,7 +1766,7 @@ int GetCodepointCount(const char *text)
     while (*ptr != '\0')
     {
         int next = 0;
-        int letter = GetCodepointNext(ptr, &next);
+        (void)GetCodepointNext(ptr, &next);
```

Comment:
I usually avoid the `(void)` at beginning when not interested in return value.

Reply:
Good point. It doesn't even suppress anything in this case. Removed the `void` cast in 814b23b367806ec9a9c2887fa0a1a8e33fa7ff5b.